Packs/CommonScripts/Scripts/SearchIncidentsV2/SearchIncidentsV2_test.py

```diff
@@ -75,6 +76,7 @@ def test_is_incident_id_valid(id_value, expected_output):
         u'id': u'1',
         u'type': u'TypeA',
         u'name': u'Phishing',
+        u'events': ['event1', 'event2', 'event3']
```

Comment:
[PEP8](https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas)
```suggestion
        u'events': ['event1', 'event2', 'event3'],
```

Reply:
it isn't actually used in the test, I'll remove it instead.